### PR TITLE
[Test/Sink] Add rosbag-based pipeline test cases for videotestsrc/audiotestsrc

### PR DESCRIPTION
**Pull request by wooksong**
Related epic: https://github.com/nnsuite/nnstreamer/issues/1059
See also: https://github.com/nnsuite/nnstreamer/issues/1121

This PR mainly concerns adding a set of utilities and test cases for rosbag-based pipeline test.

### Details ###
- [Test/Common] Add a python script for rosbag-based tests
  - ~~In the current version, comparing a raw data file containing video/x-raw with a rosbag file corresponding to the raw file is only supported. (it will be extended.)~~
  - Comparing a raw data file containing video/x-raw or audio/x-raw with a rosbag file corresponding to the raw file is supported now.
- 1) [Test/Sink] Add rosbag-based pipeline test cases
- 2) [Test/Sink] Add test cases covering audio format 
  - Raw data files created by dumping a video/x-raw frame (RGB, BGRx, and GRAY8) and 8000 audio/x-raw frames (S16LE, F32LE, and F64LE) using filesink are compared with rosbag files corresponding those raw data files.

### Changes in this PR ###
- [V2] Update test_utils.py to support audio/x-raw format
- [V2] Add est cases covering audio format 

### Test Results ###
```bash
./nnstreamer-ros/tests  (add-unittest-sink) $ ssat

==================================================

==================================================
    Test Group rosbag-based pipeline test Starts.
[PASSED] 1-1:gst-launch of case 1-1
[PASSED] 1-1:tensor_ros_sink: simple test case using raw data (video/BGRx/uint8)
[PASSED] 1-2:gst-launch of case 1-2
[PASSED] 1-2:tensor_ros_sink: simple test case using raw data (video/RGB/uint8)
[PASSED] 1-3:gst-launch of case 1-3
[PASSED] 1-3:tensor_ros_sink: simple test case using raw data (video/GRAY8/uint8)
[PASSED] 2-1:gst-launch of case 2-1
[PASSED] 2-1:tensor_ros_sink: simple test case using raw data (audio/mono/s16le/8k)
[PASSED] 2-2:gst-launch of case 2-2
[PASSED] 2-2:tensor_ros_sink: simple test case using raw data (audio/stereo/s16le/8k)
[PASSED] 2-3:gst-launch of case 2-3
[PASSED] 2-3:tensor_ros_sink: simple test case using raw data (audio/mono/f32le/8k)
[PASSED] 2-4:gst-launch of case 2-4
[PASSED] 2-4:tensor_ros_sink: simple test case using raw data (audio/stereo/f32le/8k)
[PASSED] 2-5:gst-launch of case 2-5
[PASSED] 2-5:tensor_ros_sink: simple test case using raw data (audio/mono/f64le/8k)
[PASSED] 2-6:gst-launch of case 2-6
[PASSED] 2-6:tensor_ros_sink: simple test case using raw data (audio/stereo/f64le/8k)
==================================================
[PASSED] Test Group rosbag-based pipeline test Passed

==================================================

[PASSED] tensor_ros_sink (18 passed among 18 cases)
==================================================
[PASSED] All Test Groups (1) Passed!
         TC Passed: 18 / Failed: 0
```

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

File: tests/tensor_ros_sink/runTest.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	search="ssat-api.sh"
+	source $search
+	printf "${Blue}Independent Mode${NC}
+"
+fi
+testInit "rosbag-based pipeline test"
+
+PATH_TO_PLUGIN="../../build"
+
+gst-launch-1.0 -m -v --gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/x-raw,format=BGRx,width=280,height=40,framerate=0/1 ! tee name=t ! queue ! tensor_converter silent=TRUE ! tensor_ros_sink dummy=true enable-save-rosbag=true location="simple.uint8.BGRx.bag" sync=true t. ! queue ! filesink location="simple.uint8.BGRx.log" sync=true
```
Comment (myungjoo) [inline tests/tensor_ros_sink/runTest.sh]: I recommend to use ```gstTest``` macro. It will report the result of gst-launch.

Comment (wooksong) [reply tests/tensor_ros_sink/runTest.sh]: I will apply your comment. Thanks!

**Review by myungjoo — APPROVED** [review]
LGTM!

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**